packages/cli/src/ui/types.ts

```diff
@@ -70,11 +75,13 @@ export type HistoryItemUser = HistoryItemBase & {
 export type HistoryItemGemini = HistoryItemBase & {
   type: 'gemini';
   text: string;
+  thought?: ThoughtSummary | null;
 };
 
 export type HistoryItemGeminiContent = HistoryItemBase & {
   type: 'gemini_content';
   text: string;
+  thought?: ThoughtSummary | null;
```

Comment:
interesting. wouldn't have expected this change. Curious to understand why it is needed.